src/UI/Actions.hs

```diff
@@ -1006,8 +1055,8 @@ displayThreadMails =
         -- Update the Application state with all mails found for the
         -- currently selected thread.
         dbpath <- use (asConfig . confNotmuch . nmDatabase)
-        selectedItemHelper (asMailIndex . miListOfThreads) $ \t ->
-          runExceptT (Notmuch.getThreadMessages dbpath t)
+        selectedItemHelper (asMailIndex . miListOfThreads) $ \(_, t) ->
+          runExceptT (Notmuch.getThreadMessages dbpath [t])
```

Comment:
nit: I'd prefer `Identity` here rather than `[]`.  Not because it's lower runtime overhead (though it is)
but because it more accurately expresses the intent, i.e. that there is always exactly one value in
this case.

Reply:
Aaaah I see. Thank you very much!